### PR TITLE
Show audio track names

### PR DESCRIPTION
**Pull request by arshubham**
image

Moved `get_discoverer_info` to `Utils.vala` as the same can be reused for getting subtitle langs and checking for missing plugin in https://github.com/elementary/videos/pull/155

### CODE REVIEW

**Comment by danrabbit** [comment]
Don't forget to update AppData :)

File: src/Widgets/Player/SettingsPopover.vala [hunk 1]
```diff
@@ -149,7 +147,12 @@ public class Audience.Widgets.SettingsPopover : Gtk.Popover {
         var languages_names = get_audio_track_names ();
         uint track = 1;
         playback.get_audio_streams ().foreach ((lang) => {
-            languages.append (lang, _("Track %u - %s").printf (track, languages_names.nth_data (track - 1)));
+            var audio_stream_lang = languages_names.nth_data (track - 1);
+            if (audio_stream_lang != null) {
+                languages.append (lang, _("Track %u - %s").printf (track, audio_stream_lang));
```
Comment (danrabbit) [inline src/Widgets/Player/SettingsPopover.vala]: If we can get the name of the track, do we need to show the track number? It looks like other video players only show the title.

@cassidyjames ?

Comment (cassidyjames) [reply src/Widgets/Player/SettingsPopover.vala]: @danrabbit yeah I think getting the title would be enough information, and the number is a fallback. I can't think of an instance where the track number itself is important—but admittedly I don't frequently change audio tracks in the first place.

Comment (arshubham) [reply src/Widgets/Player/SettingsPopover.vala]: I agree with you both. I just kept it same and how it show in VLC and other players.

Comment (arshubham) [reply src/Widgets/Player/SettingsPopover.vala]: Now I only show Language. See top comment.

**Pull request merged**